dali/python/nvidia/dali/eager.py

```diff
@@ -0,0 +1,242 @@
+# Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+
+import nvidia.dali.backend as _b
+import nvidia.dali.internal as _internal
+import nvidia.dali.ops as _ops
+import nvidia.dali.tensors as _tensors
+from nvidia.dali._utils.eager_util import _Classification, _transform_data_to_tensorlist
+
+
+_statefull_operators = {
```

Comment:
I wonder how we can solve the problem of having to list things manually here and due to that having to add the operator to another "registry" after implementing it.
If we add something stateful it will be exposed by mistake now. 

We can think about something for a followup.

Reply:
The backend implementation of operators seems to favor keeping this in `op_schema`, but in my opinion it would only make it worse. With this solution at least we have the whole registry in one place.